src/test/java/io/radanalytics/streaming/amqp/JavaAMQPBrokerStreamSuite.java

```diff
@@ -80,7 +83,7 @@ public void teardown() {
     @Test
     public void testAMQPReceiveSimpleBodyString() {
 
-        Function converter = new JavaAMQPBodyFunction<String>();
+        Function<Message, Option<String>> converter = new JavaAMQPBodyFunction<>();
```

Comment:
is this pertaining to the purpose of PR?

Reply:
nope, but it removes [a warning](https://travis-ci.org/radanalyticsio/streaming-amqp/builds/282785346#L2176)